splunk-otel-android/src/test/java/com/splunk/rum/PostApi29NetworkDetectorTest.java

```diff
@@ -35,121 +37,109 @@
 @Config(sdk = Build.VERSION_CODES.Q)
```

Comment:
Does it work on Android P?

Reply:
Yeah, I had missed that one. I'll change it.